src/main/java/Duke.java

```diff
@@ -1,10 +1,106 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    private static Tasks[] list_of_tasks = new Tasks[101];
+    private static int counter = 0;
+
+    public static void greet_user() {
+        System.out.println("Hello! I'm Duke \n");
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(logo);
+        System.out.println("How can i help u? \n");
+    }
+
+    public static void print_action() {
+        System.out.println("\n");
+        System.out.println("I have added this task: ");
+        System.out.println(list_of_tasks[counter - 1].toString());
+
+        System.out.println("You now have " + counter + " tasks in the list.");
+    }
+
+    public static void Todo(String description) {
+        Tasks t = new ToDo(description);
+        list_of_tasks[counter] = t;
+        counter++;
+        print_action();
+    }
+
+    public static void Deadline(String description, String by) {
+        Deadline d = new Deadline(description, by);
+        list_of_tasks[counter] = d;
+        counter++;
+        print_action();
+    }
+
+    public static void Event(String description, String start, String end) {
```

Comment:
Avoid using constructor names of other classes as method name. It can be confusing and potential source of bug. Also, consider separate classes for adding new tasks.